charts/pvc-autoresizer/templates/_helpers.tpl

```diff
@@ -60,3 +60,44 @@ Create the name of the service account to use
 {{- default "default" .Values.serviceAccount.name }}
 {{- end }}
 {{- end }}
+
+{{/*
+Create the name of the webhook service
+*/}}
+{{- define "pvc-autoresizer.webhookService" -}}
+{{- printf "%s-controller" (include "pvc-autoresizer.fullname" .) -}}
+{{- end -}}
+
+{{/*
+Create the name of the webhook cert secret
+*/}}
+{{- define "pvc-autoresizer.webhookCertSecret" -}}
```

Comment:
ditto

Reply:
It is used here
https://github.com/tropnikovvl/pvc-autoresizer/blob/main/charts/pvc-autoresizer/templates/controller/secret.yaml#L8

and here
https://github.com/tropnikovvl/pvc-autoresizer/blob/main/charts/pvc-autoresizer/templates/_helpers.tpl#L86